### PR TITLE
adding comments to guessing game

### PR DESCRIPTION
**Pull request by himanshurajora**

### CODE REVIEW

File: 2_guessing_game/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }
+        };
+
+        println!("Your guess is : {guess}");
+
+        match guess.cmp(&secret_number) {
+            Ordering::Less => println!("Too small"),
+            Ordering::Greater => println!("Too big"),
+            Ordering::Equal => {
+                println!("You win!");
```
Comment (pank1999) [inline 2_guessing_game/src/main.rs]: The `Ordering` Enum also requires comments.

File: 2_guessing_game/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            .expect("i can't  read your message");
+        let guess: u32 = match guess.trim().parse() {
+            Ok(num) => num,
+            Err(_) => {
+                println!("Fuck yu ! enter the number only");
+                continue;
+            }
+        };
+
+        println!("Your guess is : {guess}");
```
Comment (pank1999) [inline 2_guessing_game/src/main.rs]:
```rust
        println!("Your guess is : {guess}");
```
Just got to know about this method of string templating in rust. Something new!

File: 2_guessing_game/src/main.rs (hunk 1 above)
Comment (himanshurajora) [reply 2_guessing_game/src/main.rs]: sure, will do that

**Comment by himanshurajora** [comment]
super seeded by #6

**Comment by harsh-100** [comment]
If this PR is completed then , please remove WIP and also don't forget to first take pull from the main branch to take the new changes

**Comment by himanshurajora** [comment]
> If this PR is completed then , please remove WIP and also don't forget to first take pull from the main branch to take the new changes

It is already marked as closed

**Pull request closed without merging**